### PR TITLE
Replaced docs links with permalinks

### PR DESCRIPTION
**Pull request by cwarnermm**
Replaced broken & working in-product Integration management docs links with permanent links.

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx [hunk 1]
```diff
@@ -674,7 +674,7 @@ export default class PluginManagement extends AdminSettings<Props, State> {
                         values={{
                             link: (msg: React.ReactNode) => (
                                 <ExternalLink
-                                    href='https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta/'
+                                    href='https://mattermost.com/pl/default-beta-plugins/'
```
Comment (wiggin77) [inline webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: Why "beta"?

Comment (cwarnermm) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: Good question, @wiggin77. That's an existing permalink. @emdecr - Can we update this permalink to remove the "beta" part, please?

Comment (wiggin77) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: The pl resolves to `https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/` which has no `beta` in the url.

File: webapp/channels/src/components/integrations/abstract_command.jsx [hunk 2]
```diff
@@ -437,7 +437,7 @@ export default class AbstractCommand extends React.PureComponent {
                                         values={{
                                             link: (
                                                 <ExternalLink
-                                                    href='https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/#built-in-commands'
+                                                    href='https://developers.mattermost.com/integrate/slash-commands/built-in/'
```
Comment (wiggin77) [inline webapp/channels/src/components/integrations/abstract_command.jsx:440]: Is this meant to be a perma-link?

Comment (cwarnermm) [reply webapp/channels/src/components/integrations/abstract_command.jsx:440]: aha! Ideally yes. @emdecr - Can we create permalinks that point to subtopics on a page? If so, can you please create one for this topic as ``/pl/built-in-slash-commands?

File: webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx (hunk 1 above)
Comment (emdecr) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: @cwarnermm I can definitely make a switch - to clarify we want a `/pl/` link for `https://mattermost.com/pl/default-plugins/` that goes to `https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/`?

Comment (emdecr) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: @cwarnermm We have an existing `/pl/` for that one with the project we're doing with Felipe and it goes to https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta 😬 

Maybe `/pl/using-and-managing-plugins`?

Comment (cwarnermm) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx:677]: oh! We're talking about the same destination and we already have a permalink! I think I just used the wrong one in this PR. Sounds like we can safely mark the beta version as a duplicate.

File: webapp/channels/src/components/integrations/abstract_command.jsx (hunk 2 above)
Comment (cwarnermm) [reply webapp/channels/src/components/integrations/abstract_command.jsx:440]: Correct and thank you!!

File: webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx (hunk 1 above)
Comment (cwarnermm) [reply webapp/channels/src/components/admin_console/plugin_management/plugin_management.tsx]: Corrected via this PR.

File: webapp/channels/src/components/integrations/abstract_command.jsx (hunk 2 above)
Comment (cwarnermm) [reply webapp/channels/src/components/integrations/abstract_command.jsx]: Corrected via this PR.

**Comment by cwarnermm** [comment]
@wiggin77 - This PR is now ready for your review.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by fmartingr — APPROVED** [review]
LGTM. Some of these changes will overlap with #23694.

**Comment by cwarnermm** [comment]
@fmartingr - I'm wondering if there's any changes in this docs PR that you didn't handle yourself in your own Engineering PR, especially given that there's 15 files impacted and 15 files' worth of conflicts.

Do you have a smart/straightforward way to confirm any delta between this docs PR and what's going into v8.1?

**Comment by fmartingr** [comment]
@cwarnermm I have checked the conflicts locally and every link addressed in this PR was updated in mine, so this PR is safe to be closed.

**Comment by cwarnermm** [comment]
Thank you so much for the double-check, @fmartingr! I really appreciate it!

**Pull request closed without merging**